src/reducers/resourceReducer.js

```diff
@@ -22,6 +22,27 @@ export function isFetchingResource(state = initialState.isFetchingResource, acti
     }
 }
 
+export function headCategories(state = initialState.headCategories, action) {
+  switch (action.type) {
+    case types.LOAD_RESOURCE_DATA_SUCCESS:
+      const headCategories = {};
+      for (let data of action.resource) {
+        let headCat = data.headcategories.split(',');
+        headCat.forEach(group => headCategories[group] = group.trim());
```

Comment:
transcribing IRL convo: change this from group to something else; will be kinda a non-issue with refactor